### ISSUE TITLE
Uploading this file doesn't do anything

### ISSUE THREAD

**Issue by stinos** [Wirena/clang-format-configurator-v2#4]
Reg. reddit conversation, this is the file which when uploaded has no effect: no error, but also no values filled in. I checked and it's from clang-format 15, so that might be the issue since the latest supported seems to be 14, but could do with an error message then.

[clang-format.txt](https://github.com/Wirena/clang-format-configurator-v2/files/9134234/clang-format.txt)

**Comment by Wirena** [comment]
Yep, i used js exceptions incorrectly, error message didnt popup
Fixed in 59255ff8c150c9a0696881c8b14bbe82b1eb8c43
